### DISCUSSION TITLE
ubo logger entry dialogue

### DISCUSSION THREAD

**Discussion by ghajini** [uBlockOrigin/uBlock-issues#2497] [Q&A] [answered]
hii all
since upgrading to ubo 1.47 on firefox android iam having trouble viewing logger entry dialogue, it is seen half along width
as per screenshot left area is cut off, can't drag to right to see whole details
![Screenshot_20230216-000057~3](https://user-images.githubusercontent.com/20338483/219122330-1b582ca5-af74-43f7-9d65-fe5b0419b717.png)

& after rotating the screen i can't see create button
![Screenshot_20230216-001410~2](https://user-images.githubusercontent.com/20338483/219123524-85b420bf-effc-4660-8b63-9e29dd430f0c.png)

**Comment by ghost** [comment]
Bug.

**Comment by gorhill** [comment]
Meanwhile, can you scroll the whole logger horizontally?

**Reply by ghajini** [reply]
i can't scroll horizontally....like eg when clicked that entry dialogue is put extreme left and can't dragged to right on portrait mode
also if i change to landscape mode bottom area is cut like **create button** , so in order to see that button, iam not able to drag that dialogue upward

**Reply by RicoHeat** [reply]
I am having the same exact problem.. 
Fennec F-droid

**Comment by Yuki2718** [comment]
There may possibly be more serious regression in 1.47.0 tho I can't reproduce. I see some tweets claiming uBO stopped working:
```
https://twitter.com/shinsokusen/status/1626191757015912449
https://twitter.com/yuki_cre/status/1626184723151355904
https://twitter.com/terra112r/status/1626150330194100224
```

**Reply by gorhill** [reply]
Likely they are using Firefox for Android 68, I need to bump up the minimum version to 79, there is API now used which became available only in 79.

**Reply by Yuki2718** [reply]
Same for desktop?

**Reply by gorhill** [reply]
Yes, I need to bump desktop to 78, but since I don't want to have two separate extension packages for Firefox desktop and Firefox android, I will have to bump the minimum version to 79 for both.

---

Bottomline is that I don't have the resource/time to test uBO for oldest versions of browsers, I count on those minority of users using old versions to ensure the dev build work in their browser. I should have spotted that the usage of newer API required that I raised the minimum version but there is just so much I can juggle in my head, hence why I put a lot of faith in issues being spotted during development. I waited quite a long time in the last dev cycle to ensure nothing came up, and still, it's not going smoothly.

**Comment by Yuki2718** [comment]
Reload button when you temp disabled scriptlet rule (filterAuthorMode: true) has gone?

**Reply by gorhill** [reply]
You can use F5 now, or just the reload button at the top of the logger.

**Reply by Yuki2718** [reply]
No, that does not reload the tab a rule is disabled. I have to move from the logger tab.

**Reply by gorhill** [reply]
I put back the reload button in <https://github.com/gorhill/uBlock/commit/191f2f00d6f1b8a42628eba0f10ee29ac8d86b55> -- I overlooked that this specific reload button acted on the tab id associated with the event rather than the tab id associated with the dropdown list.

**Comment by ghajini** [comment]
~main issue fixed in stable build 1.47.2~
wait see following reply

**Reply by ghajini** [reply]
@gorhill 
i find this is not solved yet,i captured a video of it, have a look at it

`https://gofile.io/d/d6FBm9`

this happens when logger entries are large and entire left area is cropped

**Reply by ghost** [reply, marked as answer]
https://github.com/gorhill/uBlock/commit/7bf3f1bd2008d0f6483ca890414294714f665286
